README.md

```diff
@@ -92,47 +92,128 @@ The milestone needs testing on all platforms. Please report issues when found.
-```sh
-sudo apt update
-sudo apt install -y --no-install-recommends ninja-build cmake g++ libsdl2-dev libncurses-dev zlib1g-dev
-cmake --preset linux -D ENABLE_LOGGER=[ON|OFF]
-cmake --build --preset linux --config [Debug|Release]
-```
-
-#### Building - Linux (Fedora)
-```sh
-sudo dnf update --refresh
-sudo dnf install -y ninja-build cmake gcc-c++ SDL2-devel ncurses-devel zlib-devel
-cmake --preset linux -D ENABLE_LOGGER=[ON|OFF]
-cmake --build --preset linux --config [Debug|Release]
-```
+1. Open an appropriate command-line interpreter:
+
+    - **Windows:** Open a regular PowerShell window (we’ll switch to using a developer PowerShell later).
+    - **Other platforms:** Open a terminal window.
+
+2. Make sure that the first set of prerequisites are installed:
+
+    <details>
+    <summary><b>Windows</b></summary>
+
+    1. Make sure that your system meets the minimum requirements for either [Visual Studio 2022](https://learn.microsoft.com/en-us/visualstudio/releases/2022/system-requirements#visual-studio-2022-system-requirements) or its [Build Tools](https://learn.microsoft.com/en-us/visualstudio/releases/2022/system-requirements#microsoft-visual-studio-build-tools-2022-system-requirements). [32-bit versions of Windows are not supported.](https://learn.microsoft.com/en-us/answers/questions/1689898/does-visual-studio-build-tools-2022-support-32-bit)
+
+    2. Make sure that Git is installed:
+
+        ```pwsh
+        winget install Git.Git
+        ```
+
+    3. Restart PowerShell. This will make sure that the `git` command is on your `Path`.
+
+    4. If you don’t already have one, then create a clone of the vcpkg repo:
+
+        ```pwsh
+        git clone https://github.com/microsoft/vcpkg.git
+        ```
+
+    5. Make sure that the `VCPKG_ROOT` environment variable is set:
+
+        ```pwsh
+        setx VCPKG_ROOT 'C:\path\to\vcpkg'
+        ```
+
+    </details>
+
+    <details>
+    <summary><b>macOS</b></summary>
+
+    1. Make sure that [Xcode](https://developer.apple.com/xcode) is installed.
+
+    2. Make sure that [Homebrew](https://brew.sh) is installed.
+
+    </details>
+
+    <details>
+    <summary><b>Ubuntu</b></summary>
+
+    Run these commands:
+
+    ```sh
+    sudo apt update
+    sudo apt install -y --no-install-recommends git ninja-build cmake g++ libsdl2-dev libncurses-dev zlib1g-dev
+    ```
+
+    </details>
+
+    <details>
+    <summary><b>Fedora</b></summary>
+
+    Run these command:
+
+    ```sh
+    sudo dnf update --refresh
+    sudo dnf install -y git ninja-build cmake gcc-c++ SDL2-devel ncurses-devel zlib-devel
+    ```
+
+    </details>
+
+
+3. Clone this repo:
+
+    ```sh
+    git clone https://github.com/DescentDevelopers/Descent3
+    ```
+
+4. Change directory into the clone:
+
+    ```sh
+    cd Descent3
+    ```
+
+5. Install the rest of the prerequisites:
+
+    - **Windows:**
+
+        ```pwsh
+        powershell -ExecutionPolicy Bypass -File Install-WindowsDependencies.ps1
```

Comment:
I feel like this makes the process look more complicated than what it really is. For anyone that has ever compiled any C++ on Windows, there is literally nothing more to do than opening x64/x86 command prompt and run the 2 CMake commands selecting the preset and building, nothing more is required than a working compiler. IMO we should not burden ourselves with providing a script to install MSVC

Reply:
> I feel like this makes the process look more complicated than what it really is.

How would you recommend that I change the instructions?

> IMO we should not burden ourselves with providing a script to install MSVC.

I pushed some potential fixups for this PR [here](https://github.com/Jayman2000/Descent3-pr/tree/85899e647d921a3f134bed7cd3e81386a791e8b5?tab=readme-ov-file#building). Do you think that that version without the script is better?

Reply:
> How would you recommend that I change the instructions?

I'd suggest we start by stating requirements to build under Windows: a working VS installation that can compile C++. A link can be sufficient for that, maybe suggesting the desktop C++ dev component. This way, experienced developers can just skip it and not run a full script when they have nothing to do, and newbies can customize their installation and follow the process.

Then, keep the instructions as simple as possible. I see in #332 that you struggled with choosing the right command prompt, and to install VCPKG. I have to admit I did not follow the readme as now VCPKG is [installed with Visual Studio](https://devblogs.microsoft.com/cppblog/vcpkg-is-now-included-with-visual-studio/), so envvars are already properly set and I did not have to touch anything. Then, I just open the x86 command prompt, and run cmake with the win32 preset, build and it just works. x64 support is still experimental, we should not advertise it in the readme yet.

Reply:
Your feedback in #332 is very useful, because we don't know how many people have tried building the project, failed and quit without opening any issue. It's a challenge but providing concise AND sufficient information to build the project but crucial to keep the community and project alive.

Reply:
Visual Studio 2022, at the time of writing those windows build steps, included cmake and git in the C++ tools but not vcpkg. It was easy enough to right-click in file explorer, click Terminal, and use the command line parameters from there for vcpkg. I wasn't even aware VS2022 installed several other interpreter options. You're spot on regarding `set` and `setx` though--it does require a space instead of an equal sign.

But, `set` is a temporary use. The environment variable you set using `set` is only good for the current interpreter session. If `setx` is used, that variable is made permanent.

Reply:
I just pushed a new version of this PR. Here are the main differences between the old version and this new version:

- I removed the script and the `.vsconfig` file.
- I removed the part that tells you to manually clone vcpkg and set the `VCPKG_ROOT` environment variable. The new version of this PR relies on the fact that the “Desktop development with C++” workload sets up vcpkg for you.

I took these two suggestions:

> I'd suggest we start by stating requirements to build under Windows: a working VS installation that can compile C++. A link can be sufficient for that, maybe suggesting the desktop C++ dev component. This way, experienced developers can just skip it and not run a full script when they have nothing to do, and newbies can customize their installation and follow the process.

> Given VS2022 now includes vcpkg, though, that's irrelevant. That section could be simplified to `Requires Visual Studio 2022 and the Desktop Development with C++ workload` and it would be fine.

and implemented something that’s similar to them. In addition to telling users that they need Visual Studio 2022 with that workload, the new version of this PR gives users two commands that they can run if they don’t already have Git and Visual Studio 2022 with that workload. I provided those two commands because I wanted to have feature parity with Linux. On Ubuntu and Fedora, users can simply copy and paste two commands in order to make sure that they have everything that they need in order to build Descent 3. I wanted Windows users to have that same convenience. That being said, I can replace those two commands with a links for how to set up Git and Visual Studio 2022 if you guys think that that would be better.